lib/Backend/BackwardPass.cpp

```diff
@@ -8710,6 +8710,12 @@ BackwardPass::VerifyByteCodeUpwardExposed(BasicBlock* block, Func* func, BVSpars
         // We've collected the Backward bytecodeUpwardExposeUses for nothing, oh well.
         if (this->func->hasBailout)
         {
+            if (instr->GetPrevRealInstrOrLabel()->GetByteCodeOffset() == instr->GetByteCodeOffset())
```

Comment:
Shouldn't there be a bytecodeUse for s51 and shouldn't s42 be marked as jit optimized?